src/test/java/org/java_websocket/util/CharsetfunctionsTest.java

```diff
@@ -0,0 +1,57 @@
+package org.java_websocket.util;
+
+import org.java_websocket.exceptions.InvalidDataException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.nio.ByteBuffer;
+
+public class CharsetfunctionsTest {
+
+  @Test
+  public void testAsciiBytes() {
+    Assert.assertArrayEquals(new byte[] {102, 111, 111}, Charsetfunctions.asciiBytes("foo"));
+  }
+
+  @Test
+  public void testStringUtf8Byte() throws InvalidDataException {
```

Comment:
Isn't this the same as the next test case (`testStringUtf8ByteBuffer()`)?

Reply:
Thanks this test has been removed